wire-runtime-swift/src/main/swift/propertyWrappers/Heap.swift

```diff
@@ -22,6 +22,7 @@ import Foundation
  Use of this wrapper is required on large protobuf messages because large (and especially
```

Comment:
Can we make sure this comment gets copied onto CopyOnWrite before this is fully removed (or now)?